setup.py

```diff
@@ -6,7 +6,7 @@
 import warnings
 
 VERSION = 'undefined'
-install_requires = ['six', 'pyprind']
+install_requires = ['six', 'pyprind', 'pyparsing==2.4.7']
```

Comment:
what uses pyparsing? is this still needed?

Reply:
Not used, removed as a requirement